src/Commands/CatchStartCommand.ts

```diff
@@ -0,0 +1,56 @@
+import { Command } from "./Command";
+import { CommandResult } from "./CommandResult";
+import { LineResults } from "./LineResults";
+import { Parameter } from "./Parameters/Parameter";
+import { SingleParameter } from "./Parameters/SingleParameter";
+
+/**
+ * A command for the beginning of a catch block.
+ */
+export class CatchStartCommand extends Command {
+    /**
+     * Information on parameters this command takes in.
+     */
+    private static parameters: Parameter[] = [
+        new SingleParameter("exception", "Target exception.", true),
+        new SingleParameter("alias", "Alias for target exception.", true)
```

Comment:
The "alias", as far as I can tell, is required for TypeScript, and therefore i thought it should be required for GLS. Correct me if i'm wrong on that.